command/pr.go

```diff
@@ -116,7 +116,7 @@ func prStatus(cmd *cobra.Command, args []string) error {
 	repoOverride, _ := cmd.Flags().GetString("repo")
 	currentPRNumber, currentPRHeadRef, err := prSelectorForCurrentBranch(ctx, baseRepo)
 
-	if err != nil && repoOverride == "" && err.Error() != "git: not on any branch" {
+	if err != nil && repoOverride == "" && err != git.ErrNotOnAnyBranch {
```

Comment:
To cover cases when `ErrNotOnAnyBranch` was wrapped in another error, the equality operator should be replaced with `errors.Is(err, git.ErrNotOnAnyBranch)`

Reply:
Oh awesome, I didn't know about this function, but it seems very useful. Will fix.